packages/mc-scripts/src/utils/get-session-data.js

```diff
@@ -0,0 +1,26 @@
+const fetch = require('node-fetch');
+const cookie = require('cookie');
+
+const getSessionData = async ({ email, password, mcApiUrl }) => {
+  const response = await fetch(`${mcApiUrl}/tokens`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ email, password }),
+  });
+
+  if (!response.ok) {
+    const rawResponseError = await response.text();
+    const errorMessage = JSON.parse(rawResponseError)['message'];
+    throw new Error(errorMessage);
```

Comment:
What I meant was to safely JSON parse, like this:

```suggestion
    const text = await response.text();
    let parsed;
    try {
      parsed = JSON.parse(text);
    } catch (error) {
    }
    const errorMessage = parsed ? parsed.message : text
    throw new Error(errorMessage);
```